app/errors.js

```diff
@@ -0,0 +1,15 @@
+exports.handlePostgreErrors = (err, req, res, next) => {
```

Comment:
Nice error handling. But you have added code here that handles a psql error that you server doesn't generate yet. Please only add code that you need add functionality to pass tests.